sdk/js-connect/core/base/src/constants/contracts/relayer.ts

```diff
@@ -0,0 +1,37 @@
+import { MapLevel } from "../../utils";
+import { Network } from "../networks";
+import { Chain } from "../chains";
+
+export const relayerContracts = [[
+  "Mainnet", [
+    ["Ethereum",  "0x27428DD2d3DD32A4D7f7C497eAaa23130d894911"],
```

Comment:
I'm guessing you copy pastad all this and they don't need to be individually verified?

Reply:
yes copy/pasta'd but probably makes sense to check a few